### PR TITLE
Add parameter to enable logging in connection string / DSN

### PR DESCRIPTION
**Pull request by SimbaGithub**
### Description
Add a connection property 'Tracing' to enable logging via connection string or DSN.

### Checklist
- [x] Code compiles correctly
- [x] Run ``make fmt`` to fix inconsistent formats
- [x] Run ``make lint`` to get lint errors and fix all of them
- [ ] Created tests which fail without the change (if possible)
- [x] All tests passing
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

File: doc.go [hunk 1]
```diff
@@ -112,6 +112,8 @@ The following connection parameters are supported:
 	* validateDefaultParameters: true by default. Set to false to disable checks on existence and privileges check for
 								 Database, Schema, Warehouse and Role when setting up the connection
 
+    * tracing: Specifies the logging level to be used. Set to error by default.
```
Comment (sfc-gh-mheavin) [inline doc.go:115]: What other levels does the parameter support?

**Review by sfc-gh-mheavin — COMMENTED** [review]
Please clarify the valid values

File: dsn.go [hunk 2]
```diff
@@ -604,6 +609,8 @@ func parseDSNParams(cfg *Config, params string) (err error) {
 			} else {
 				cfg.ValidateDefaultParameters = ConfigBoolFalse
 			}
+		case "tracing":
```
Comment (sfc-gh-hchaturvedi) [inline dsn.go:612]: Where are we setting the default to error?

Comment (SimbaGithub) [reply dsn.go:612]: In driver.go, https://github.com/snowflakedb/gosnowflake/blob/master/driver.go#L55

File: doc.go (hunk 1 above)
Comment (SimbaGithub) [reply doc.go:115]: Updated to include valid values.

**Review by sfc-gh-mheavin — APPROVED** [review]

**Review by sfc-gh-hchaturvedi — APPROVED** [review]
LGTM

**Pull request merged**